Source: Scribus Bugtracker
Upstream: http://bugs.scribus.net/view.php?id=9005
Reason: Fix automagic dependencies on aspell and podofo

diff -Naur scribus-1.3.6/CMakeLists.txt scribus-1.3.6.new/CMakeLists.txt
--- scribus-1.3.6/CMakeLists.txt	2010-03-15 23:01:54.000000000 +0100
+++ scribus-1.3.6.new/CMakeLists.txt	2010-04-09 21:30:54.557028618 +0200
@@ -645,24 +645,29 @@
 #>>FontConfig
 
 #<<ASPELL for Speelling support
-FIND_PACKAGE(ASPELL)
-IF (ASPELL_FOUND)
-  MESSAGE("ASpell Found OK")
-  SET(HAVE_ASPELL 1)
-ELSE(ASPELL_FOUND)
-  MESSAGE("ASpell or its developer libraries NOT found - Disabling support for spell checking")
-ENDIF(ASPELL_FOUND)
+OPTION(WITH_ASPELL "Enable support for spell checking" ON)
+IF (WITH_ASPELL)
+  FIND_PACKAGE(ASPELL)
+  IF (ASPELL_FOUND)
+    MESSAGE("ASpell Found OK")
+    SET(HAVE_ASPELL 1)
+  ELSE (ASPELL_FOUND)
+    MESSAGE("ASpell or its developer libraries NOT found - Disabling support for spell checking")
+  ENDIF (ASPELL_FOUND)
+ENDIF (WITH_ASPELL)
 #>>ASPELL for Speelling support
 
-
 #<<PoDoFo for AI PDF import
-FIND_PACKAGE(LIBPODOFO)
-IF(LIBPODOFO_FOUND)
-  MESSAGE("PoDoFo found OK")
-  SET(HAVE_PODOFO 1)
-ELSE(LIBPODOFO_FOUND)
-  MESSAGE("PoDoFo NOT found - Disabling support for PDF embedded in AI")
-ENDIF(LIBPODOFO_FOUND)
+OPTION(WITH_PODOFO "Enable support for PDF embedded in AI" ON)
+IF (WITH_PODOFO)
+  FIND_PACKAGE(LIBPODOFO)
+  IF (LIBPODOFO_FOUND)
+    MESSAGE("PoDoFo found OK")
+    SET(HAVE_PODOFO 1)
+  ELSE (LIBPODOFO_FOUND)
+    MESSAGE("PoDoFo NOT found - Disabling support for PDF embedded in AI")
+  ENDIF (LIBPODOFO_FOUND)
+ENDIF (WITH_PODOFO)
 #>>PoDoFo for AI PDF import
 
 ##############################################################################################################
